devel/juno-1/bootstrap.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+set -o errexit
+set -o pipefail
+
+CLEANUP=${CLEANUP:-"0"}
+
+GREEN='\e[0;32m'
+RESET='\e[0m'
+RED='\e[0;31m'
+
+function log {
```

Comment:
This does not work on mac, i get: 

```
\e[0;32m /-------------------------------------------------------------------------------------- \e[0m
\e[0;32m - Setting up working directory \e[0m
```

Reply:
What is your shell ? I can just remove coloring. This is just eye candy.

Reply:
Im using stock bash